src/inc/msquic_posix.h

```diff
@@ -97,6 +97,10 @@ inline ENUMTYPE &operator ^= (ENUMTYPE &a, ENUMTYPE b) throw() { return (ENUMTYP
 #define ENOKEY 126
 #endif // ENOKEY
 
+#ifndef ETIME // undefined on FreeBSD
+#define ETIME ETIMEDOUT
```

Comment:
We need a different value here so we can differentiate `QUIC_STATUS_CONNECTION_TIMEOUT` and `QUIC_STATUS_CONNECTION_IDLE` (see below).